validator/internal/validator_service_mock.go

```diff
@@ -6,12 +6,11 @@ package internal
 
 import (
 	context "context"
-	reflect "reflect"
-
 	gomock "github.com/golang/mock/gomock"
 	v1 "github.com/prysmaticlabs/prysm/proto/beacon/rpc/v1"
 	grpc "google.golang.org/grpc"
 	metadata "google.golang.org/grpc/metadata"
+	reflect "reflect"
```

Comment:
how did this happen? can we revert this? and for other mock files as well

Reply:
I had to run the script update-mockgen due to an error in the tests. This script is supposed to be executed whenever there is a change/addition to rpc calls right? 
The script added the "reflect" statement and the tests started working again. I can remove this line  but it would probably prevent the mocks from working.

Reply:
i mean to move it to it's original position since this not a 3rd party imported package, its now at the wrong place

Reply:
Sure. I'll do it.